flambdatest/mlexamples/unused.ml

```diff
@@ -0,0 +1,7 @@
+
+let foo a b = a + b [@@inline never]
```

Comment:
You can add `[@@local never]` to forbid simplification of single use functions

Reply:
Thanks, I forgot this applied even with flambda2.